src/stactools/sentinel1/__init__.py

```diff
@@ -12,4 +12,4 @@ def register_plugin(registry: Registry):
     registry.register_subcommand(create_sentinel1_command)
 
 
-__version__ = '0.1.0'
+__version__ = '0.2.0'
```

Comment:
@maximlamare i'm hoping to merge this and release v0.2.0 so that the GRD subcommand is available via pypi

Reply:
OK, do you need anything done from my side?